docker/README.md

```diff
@@ -8,6 +8,7 @@ Miaou development environment using Docker
 ### Linux
 
  - [Docker and Docker Compose](https://docs.docker.com/engine/installation/linux/)
+ - Docker needs Hyper-V activated and purpose to activate it itself (restart needed). Beware : activating Hyper-V disables any other virtualisation softwares.
```

Comment:
AFAIK, Hyper-V has nothing to do with Linux, this is Windows only related, I don't think you want to put that in this section

Besides, with "Docker for Windows/Mac“, this is now trivial to use docker in "any" environment. So I don't think the whole `Requirements` section is still a good thing to have and maintain